test/unit/api/wide/load/arithmetic/load.hpp

```diff
@@ -293,3 +295,37 @@ TTS_CASE_TPL("Check load from range for wide", EVE_TYPE )
   T from_begin_end(ref_ptr.begin(), ref_ptr.end());
   TTS_EQUAL(from_begin_end, ref);
 }
+
+TTS_CASE_TPL("Check load unsafe, wide", EVE_TYPE)
+{
+  eve::element_type_t<T> x = 123;
+
+  {
+    auto ptr = eve::previous_aligned_address(
+      &x, eve::lane<sizeof(x) * T::static_size>);
```

Comment:
This doesn't work for smaller wides, and probably for bigger wides too. A bit annoying that we can't just `load` a wide. oh well.